src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FileReaderImpl;
+import core.basesyntax.db.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.DataConverterImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.operations.BalanceOperation;
+import core.basesyntax.service.operations.OperationHandler;
+import core.basesyntax.service.operations.PurchaseOperation;
+import core.basesyntax.service.operations.ReportGenerator;
+import core.basesyntax.service.operations.ReportGeneratorImpl;
+import core.basesyntax.service.operations.ReturnOperation;
+import core.basesyntax.service.operations.SupplyOperation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.io.FileReader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private final FileReader fileReader;

```

Comment:
The private FileReader 'fileReader' is declared but never used. Consider removing it if it's not needed.

Reply:
Actually it's used just below,